evm-contracts/src/v0.7/dev/Operator.sol

```diff
@@ -70,8 +72,8 @@ contract Operator is
     bytes32 indexed requestId
   );
 
-  event ForwarderChanged(
-    OperatorForwarder indexed addr
+  event OwnershipAccepted(
```

Comment:
Could we call this `ForwarderOwnershipAccepted`? Otherwise, it could cause confusion with the Ownership of this contract.

Reply:
I renamed it to `OwnableContractAccepted` because the Operator could hypothetically own other types of contracts in the future too. No reason we really have to lock that part down to only forwarders.